### DISCUSSION TITLE
time_to_run variable invalid

### DISCUSSION THREAD

**Discussion by dapperdivers** [meisnate12/Plex-Meta-Manager-Configs#32] [General]
Getting an error with the time_to_run variable invalid.  I did some googling to see if anyone else had this issue but came up short.  

Any thoughts?  

Appreciate the time :)

**Comment by meisnate12** [comment]
what was the exact error?

**Comment by dapperdivers** [comment]
I'm really sorry, I have become what I despise.  This is an awful bug report, and even I can't remember how to replicate it at this point.  I was running locally on my plex server and ran into this when trying to run the command. 

I ended up hosting this in docker and all of my issues went away.. 

I'll try to host locally again and see if I can update with more information. 



Thank you for your time, and sorry for the bad bug report.  I'd say lets close this one, and I can open a better one once I replicate

**Reply by meisnate12** [reply]
haha thats fine sorry i took so long to see it i dont see stuff in the configs repo all the time
